engine/access/rest/models/event.go

```diff
@@ -40,9 +40,11 @@ type BlocksEvents []BlockEvents
 
 func (b *BlocksEvents) Build(blocksEvents []flow.BlockEvents) {
 	evs := make([]BlockEvents, 0)
-	for _, ev := range blocksEvents {
-		// don't include blocks without events
-		if len(ev.Events) == 0 {
+	for i, ev := range blocksEvents {
+		// don't include blocks without events, except for the last block
+		// always include the last block so clients know which was the last block processed event
+		// when it doesn't contain any events
+		if len(ev.Events) == 0 && i < len(blocksEvents)-1 {
```

Comment:
I opted to skip all but the last block to reduce data overhead. Given grpc returns results for all blocks, I'm not totally convinced it's worth the discrepancy.

@sideninja @JeffreyDoyle what do you think?

Reply:
Basically both ways have pros/cons. This one optimizes the size but requires additional explanation, the other one is bigger but more intuitive. I think it's fine going this way but we MUST then update the docs and explain this.

Reply:
yea, that's fair. For simplicity, I'll update it to return an response for all blocks so it's consistent with the gRPC apis.